pyscf/nao/pdos.py

```diff
@@ -53,3 +57,40 @@ def pdos(mf, zomegas, nkpoints=1):
     pdos[j] = gdos(mf, zomegas, mask, over, nkpoints)
 
   return pdos
+
+#
+# Example of plotting DOS calculated by GW calculation.
```

Comment:
Why the example here? Could you add a test in the subdirectory tests for this example?

Reply:
I just follow the criterion which is done for example in mf.py. Overall, I will add into the test subdirectory.

Reply:
#over = mf.hsx.s4_csr.toarray()
  if hasattr(mf, 'hsx') :
    over = mf.hsx.s4_csr.toarray()
  else:
    over = mf.overlap_lil().toarray()

because gw has no attribute as hsx.